templates/zerver/api/writing-bots.md

```diff
@@ -192,8 +190,6 @@ handles user message.
 
 * bot_handler - used to interact with the server, e.g. to send a message
 
-#### Return values
-
```

Comment:
The changes to this file seem to be in error; I'm going to undo them.  It's well worth doing a hand-review of PRs after preparing them, since that's how I usually catch mistakes like this when I make them myself :)

Reply:
Ouch! The bash script that compared the diffs did so only for the endpoints, and so I didn't notice this error. I'll also pass my PRs through a hand-review as well from now on :)